### DISCUSSION TITLE
Linear Advance

### DISCUSSION THREAD

**Discussion by Pridanc** [mriscoc/Ender3V2S1#1284] [General]
Miguel, again, love the Mini 3 software as it works perfectly.  Question, and you know I'm ignorant so bear with me, do I need to go into the Firmware you've provided to activate Linear Advance or is this ready for my to use if I add the LA feature to my slicer?  TYVM, PDC

**Comment by mriscoc** [comment]
For be able to use Linear Advance you need a firmware compiled with LA enabled, currently only the firmwares with "LA" in the filename have it enabled: https://github.com/mriscoc/Special_Configurations/releases/tag/LinearAdvance
